routingtable/vrf/vrf_registry.go

```diff
@@ -59,12 +59,15 @@ func (r *VRFRegistry) UnregisterVRF(v *VRF) {
 	delete(r.vrfs, v.routeDistinguisher)
 }
 
-// UnregisterAll unregisters all VRFs
-func (r *VRFRegistry) UnregisterAll() {
+// DisposeAll dosposes all VRFs
+func (r *VRFRegistry) DisposeAll() {
```

Comment:
Maybe call this `DestroyAll` or rename the method of the interface to `Dispose`?

Reply:
Done.